src/DurableTask.ServiceBus/Tracking/AzureTableOrchestrationHistoryEventEntity.cs

```diff
@@ -93,8 +93,31 @@ public AzureTableOrchestrationHistoryEventEntity()
         /// <summary>
         /// Gets or set the history event detail for the tracking entity
         /// </summary>
+        [IgnoreDataMember]
```

Comment:
nit - let's add an inline comment documenting where this data will come from
```suggestion
        [IgnoreDataMember] // see HistoryEventJson
```